README.md

```diff
@@ -164,6 +164,19 @@ In order to delete artifacts on the S3 Bucket, you would have to add the propert
 `-Dio.jenkins.plugins.artifact_manager_jclouds.s3.S3BlobStoreConfig.deleteStashes=true`  to your Jenkins JVM properties
 , if it is not set the stash will not be deleted from S3 when the corresponding build is deleted.
 
+# AWS Credentials
+
+Artifact Manager on S3 plugin needs an AWS credentials in order to access to the S3 Bucket, you can select one on the
+configuration page. If you do not select any AWS credential and keep the "" dropdown on the option "IAM instance Profile/user AWS configuration"
+Artifact Manager on S3 plugin would try to use the IAM instance profile credentials of the Jenkins host, or user AWS configuration (~/.aws). 
+
+![](images/configue-credentials.png)
+
+If you use a regular Key/Secret AWS Credentials you can set the token duration by adding the property 
+`-Dio.jenkins.plugins.artifact_manager_jclouds.s3.S3BlobStoreConfig.sessionDuration` to the Jenkins JVM properties, 
+the default value is one hour. However if you set and IAM Role to assume on the AWS credential, 
+the token duration is always one hour, this parameter does not change the duration in AWS Credentials Plugin settings.
```

Comment:
the generated URL is not valid and you can not upload/download the artifacts, in case you have problems because your URLs expired you could change the duration of tokens. Maybe we should make the same for the instance profile tokens and credentials with IAM roles.

Reply:
Not sure I am following. Jenkins is interpreting this property because it is creating a token with some expiration time, right? And if the token is only good for one hour, presumably Jenkins is creating a new token some other time. So under which exact conditions would the token be expired? Please put answer in the README.

Reply:
I was thinking about that all the URLs generated are consumed but Jenkins just before generated them, the only case that you have an expired URL it is if you try to reuse the URL generated one hour after generated it, for example from the Chrome downloads screen, I will try to explain it in the README